selinux-contexts.ks.in

```diff
@@ -0,0 +1,36 @@
+#test name: selinux-contexts
+%ksappend repos/default.ks
+
+install
+
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+
+clearpart --all --initlabel
+autopart
+
+keyboard us
+lang en
+timezone America/New_York
```

Comment:
These look like defaults. Please, consider to use fragments instead.

Reply:
Interesting, I did not see the other tests use them... much. Changed, thank you.